include/swift/AST/AvailabilitySpec.h

```diff
@@ -137,6 +140,43 @@ class LanguageVersionConstraintAvailabilitySpec : public AvailabilitySpec {
   }
 };
 
+/// \brief An availability specification that guards execution based on the
+/// compile-time swiftpm manifest version, e.g., swiftpm-manifest-version >= 3.0.1.
+class SwiftPMManifestVersionConstraintAvailabilitySpec : public AvailabilitySpec {
```

Comment:
Is there an advantage to keeping this separate from LanguageVersionConstraintAvailabilitySpec?

Also, please don't use `\brief`. We just got rid of all those.